### PR TITLE
Introduce maintainer processes

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Introduce maintainer processes with security requirements
| Fixed ticket? |

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

**Review by PululuK — DISMISSED** [review]
@matks are these recommendations or requirements?

If these are the requirements, does that mean that the candidate must meet these points before having his application approved? So how can we verify this?

**Comment by matks** [comment]
> @matks are these recommendations or requirements?

Recommendations for now. It could become requirements in the future.

> If these are the requirements, does that mean that the candidate must meet these points before having his application approved? So how can we verify this?

I dont think we need to be so strict. What is important here is to highlight that maintainer rights on an organization makes the account an important asset not to be underestimated. [Some people go even further](https://twitter.com/TobiasNyholm/status/1315355043185537024) 😅 .

Even if tomorrow these become hard requirements I dont think we need maintainer to meet these points before submitting application and verify it. A ill-intentioned maintainer candidate could enable 2FA then disable it later if he wanted to trick us.

One of the important parts of being part of a maintainer team is trust 😊 . We decided through a vote that you and @kpodemski have earned our trust and could join us as maintainers. If we start verifying that indeed you enabled 2FA even though you said it, it means the trust is gone and the situation is way worse than it looks. We will not verify informations you provide us unless a very nasty situation appears.

**Comment by PululuK** [comment]
> Recommendations for now. It could become requirements in the future.
> 

It does not match your PR description. Could you update it then to avoid ambiguities?

File: src/content/1.7/project/maintainers-guide/processes.md
Comment (PululuK) [inline src/content/1.7/project/maintainers-guide/processes.md:7]:
```suggestion
## Security recommendations
```

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matks

**Pull request merged**
